master/getting-started/mesos/tutorials/connecting-tasks.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Connecting to Tasks
+---
+
+Calico assigns each mesos task a uniquely addressable IP address.
```

Comment:
Maybe "When Calico is used to network Mesos tasks, each task is assigned a uniquely addressable IP address"